index.rst

```diff
@@ -68,8 +68,10 @@ This could still be achieved using a similar solution or relying on a service li
 Service Architecture
 ====================
 
-Catcher would be a service built on top of existing influxDB and LSP functionality, with ancillary support from LOVE, the OCPS backend and LSP infrastucture.
-It will mostly rely on `flux`_ scripts and `influxDB task`_ to execute operations based on data streams from the EFD.
+existing RSP functionalities... 
+
+Catcher would be a service built on top of existing RSP functionalities, with ancillary support from LOVE, the OCPS backend and RSP infrastucture (InfluxDB is part of Sasquatch which is part of the RSP).
```

Comment:
We should rephrase this sentence to something like this:

Some of the RSP functionalities mentioned here are still under development but can potentially be used for the Catcher use case. Changes in LOVE and OCPS may also be required to interface with the Catcher.

Reply:
I rephrased it a bit but didn't used your proposed text. Although LOVE will require changes, I don't expect any change in OCPS.